src/pages/user/Profile.test.js

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+// import configureStore from 'redux-mock-store';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Profile } from './Profile';
+import { withMemoryRouter } from 'utils';
+
+describe('<Profile />', () => {
+  // const mockStore = configureStore();
+
+  const match = { params: { id: 1 }, path: '/user/profile' };
+  const collection = {
+    name: 'mixn',
+    userId: 10,
+    location: 'WhyNotMinot, CAN',
+    bio: 'FA Cheez Whiz is my favorite.',
+    url: 'itsfreezing.com',
+    gravatar: 'xxx'
+  };
+  const actions = {
+    requestCurrentUserProfile: jest.fn(),
+    requestProfile: jest.fn(),
+    updateProfile: jest.fn()
+  };
+  /* const store = mockStore({
+    profile: initialProfileState,
+    application: initialAppState,
+    match
+  });*/
+
+  const props = {
+    match,
+    collection,
+    actions,
+    editting: false,
+    loggedIn: true,
+    currentUser: true
+  };
+  const RoutedProfile = withMemoryRouter(Profile);
+
+  it('renders current user profile correctly', () => {
+    const component = renderer.create(<RoutedProfile {...props} />);
```

Comment:
Could call .toJSON() here so it's `const tree = renderer.create(...).toJSON();` and then you can `expect(tree).toMatchSnapshot()`